### PR TITLE
Upgrade mux container to Bullseye

### PR DESCRIPTION
**Pull request by zjswhhh**
sign-off: Jing Zhang zhangjing@microsoft.com

#### Why I did it
As part of the process moving containers from buster to bullseye.

#### How I did it
1. change base image from buster to bullseye. 
2. remove unused addition to orchagent run options 

#### How to verify it
Tested building locally. 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

File: dockers/docker-mux/Dockerfile.j2 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% from "dockers/dockerfile-macros.j2" import install_debian_packages, install_python_wheels, copy_files %}
-FROM docker-config-engine-buster
+FROM docker-config-engine-bulleyes
```
Comment (lguohan) [inline dockers/docker-mux/Dockerfile.j2:2]: how is this tested? is it bullseye or bulleyes?

Comment (zjswhhh) [reply dockers/docker-mux/Dockerfile.j2:2]: Converted to draft. This PR is still in progress.

**Pull request merged**